.golangci.yml

```diff
@@ -0,0 +1,118 @@
+run:
+  timeout: 10m
+  skip-files:
+    - "zz_generated.*.go"
+    - ".*_test.go"
```

Comment:
Woudn't skip test files

Reply:
I would skip tests in this first phase.

Reply:
Is it there any specific reason? Do you expect those to be characterized by more linting errors or is it just a matter of reducing the amount of files to be fixed?